curves/bls12_377/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.4.0"
 authors = [ "arkworks contributors" ]
 description = "The BLS12-377 pairing-friendly elliptic curve"
 homepage = "https://arkworks.rs"
-repository = "https://github.com/arkworks-rs/curves"
+repository = "https://github.com/arkworks-rs/algebra/tree/master/curves"
```

Comment:
The repository is just `https://github.com/arkworks-rs/algebra`, rather than the path